<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWindow</class>
 <widget class="QMainWindow" name="LauncherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LauncherWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralWidget {
  background-image: url(:/resources/images/background.jpg);
  bacoground-size: cover;
  border-style: solid;
  border-width: 12px;
  border-color: rgba(220, 255, 240, 0.6);
  border-radius: 12px;
}

</string>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Church Text 'Shaded'</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#labelTitle {
  font: 64pt &quot;Church Text 'Shaded'&quot;;
  color: rgb(20,20,20);
  font-size: 64px;
}
</string>
    </property>
    <property name="text">
     <string>Cloud Land</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameLogin {
  background-color: rgba(200, 200, 200, 0.2);
  border-style: double;
  border-width: 4px;
  border-radius: 12px;
  border-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.477, y2:0.448773, stop:0 rgba(138, 186, 187, 0.8), stop:1 rgba(224, 248, 248, 0.4));
}
QLineEdit {
  font: 16pt &quot;Arial&quot;;
  background-color: rgba(248, 248, 248, 0.6);
}
QLabel {
  font-family: &quot;Church Text 'Shaded'&quot;;
  font-size: 12pt;
  color: #eee;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="textEmail">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNoPredictiveText</set>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>E-Mail</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#labelLogin {
  color: #eee;
  font-size: 32pt;
}
</string>
     </property>
     <property name="text">
      <string>L o g i n</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textPassword">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLogin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#buttonLogin {
  font: 24pt &quot;Arial&quot;;
  color: rgb(242, 255, 255);
  border-style: solid;
  border-width: 2px;
  border-radius: 4px;
  border-color: rgba(226, 253, 255, 0.9)
}
#buttonLogin:pressed {
  background-color: rgba(10, 0, 113, 0.8);
}
</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#labelStatus {
  font-family: &quot;Arial&quot;;
  color: #eee;
  font-size: 24px;
}
</string>
     </property>
     <property name="text">
      <string>Welcome to Cloud Land! </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>E-Mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRegister">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#buttonRegister {
  font: 8pt &quot;Arial&quot;;
  color: rgb(242, 255, 255);
  border-style: solid;
  border-width: 2px;
  border-radius: 4px;
  border-color: rgba(226, 253, 255, 0.9)
}
#buttonRegister:pressed {
  background-color: rgba(10, 0, 113, 0.8);
}
</string>
     </property>
     <property name="text">
      <string>Register (FREE)</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
